### DISCUSSION TITLE
Maps with size greater than 25MB

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#1871] [Q&A] [answered]
Hello.

I want to add a map at the municipal level, but when generating the geojson the file exceeds 25MB and it won't let me load it.

I divided it into several parts and uploaded them, but when viewing the map it only recognizes one file.

In this case what should I do?

Thank you

**Comment by brockfanning** [comment, marked as answer]
@SandraEliz My recommendation is to use a smaller version of that GeoJSON file. If users have to download a 25MB file, it will have a negative effect on performance -- particularly on mobile, where downloading a 25MB can take a noticeable amount of time.

There are several downloadable tools available to do this, but I looked around for an online option and found this tutorial: https://www.statsilk.com/maps/simplify-map-reducing-file-size-and-loading-time

I would try to make the GeoJSON file as small as possible, preferably below 1MB.

**Reply by SandraEliz** [reply]
Hello

I have already reduced the map, when loading it on the platform it appears like this:
![image](https://user-images.githubusercontent.com/117297044/201996390-9fbe9a72-8371-4619-a4cd-aa5072faaddb.png)

**Reply by brockfanning** [reply]
@SandraEliz That looks like the map is not displaying at all - only the "tile images". I would suspect there is a javascript error happening in the console. I would be happy to take a look if you could post a link, but I can't tell anything more from the screenshot.

**Comment by SandraEliz** [comment]
Hi

This is what it shows in the console

![image](https://user-images.githubusercontent.com/117297044/202247552-6f69119c-e323-4fc7-a185-d7c84cc7ab64.png)

**Reply by brockfanning** [reply]
@SandraEliz Ok, the "Geometry exceeds allowed extent" message makes me think that the GeoJSON file is still too large. How large is the GeoJSON file at this time?

**Reply by SandraEliz** [reply]
6.53MB

I already reduced it to 3.6 MB and it doesn't accept it either.

I think it is the smallest thing that can be generated

**Reply by brockfanning** [reply]
@SandraEliz There may be some problem with the geojson file. I can't say for sure, without seeing the file.

**Comment by SandraEliz** [comment]
[municipiosReducidotxt.txt](https://github.com/open-sdg/open-sdg/files/10026161/municipiosReducidotxt.txt)

**Reply by brockfanning** [reply]
@SandraEliz I would recommend using mapshaper.org to "simplify" the file.

1. Make sure the filename ends in ".json"
2. Import it into mapshaper.org
3. Press "Simplify"
4. Click "Apply"
5. Drag the percentage over to about 10%
6. Click "Export"

![chrome-capture-2022-10-16](https://user-images.githubusercontent.com/1319083/202312272-60b5482f-f934-4922-ac46-0dd69d9f1276.gif)

**Reply by SandraMarq** [reply]
Hello.

Finally the map could be loaded.

I really appreciate his help.
